Library/Homebrew/rubocops/cask/ast/stanza.rb

```diff
@@ -29,7 +28,7 @@ def source_range
         end
 
         def source_range_with_comments
-          comments.reduce(source_range) do |range, comment|
+          stanza_comments(stanza_node).reduce(source_range) do |range, comment|
```

Comment:
Oh, of course this is an infinite loop, `stanza_comments` calls `comments_hash` which calls `stanza_comments`, and so on.

Reply:
🤔 Any ideas here?